src/main/python/hydro_sphere/vm_files/add_grouping.sh

```diff
@@ -22,6 +22,15 @@ echo "60mins" | sudo tee /etc/mesos-slave/gc_delay
 # Optionally you can change the mesos work directory from /tmp
 echo "/opt/mesos" | sudo tee /etc/mesos-slave/work_dir
 
+# Increase max number of memory map areas for processes
+# Full doc
+# https://www.kernel.org/doc/Documentation/sysctl/vm.txt
+echo "600000" | sudo tee /proc/sys/vm/max_map_count
```

Comment:
This is likely not going to be persistent across boot.
Need to make it persistent.